tools/expotools/src/commands/AndroidBuildPackages.ts

```diff
@@ -0,0 +1,345 @@
+  for (const filename of filenames) {
+    await fs.writeFile(filename, savedFiles[filename]);
+  }
+}
+
+async function _commentWhenDistributing(filenames: string[]): Promise<void> {
+  for (const filename of filenames) {
+    await _regexFileAsync(
+      filename,
+      `// WHEN_DISTRIBUTING_REMOVE_FROM_HERE`,
+      '/* WHEN_DISTRIBUTING_REMOVE_FROM_HERE'
+    );
+    await _regexFileAsync(
+      filename,
+      `// WHEN_DISTRIBUTING_REMOVE_TO_HERE`,
+      'WHEN_DISTRIBUTING_REMOVE_TO_HERE */'
+    );
+  }
+}
+
+async function _uncommentWhenDistributing(filenames: string[]): Promise<void> {
+  for (const filename of filenames) {
+    await _regexFileAsync(filename, '/* UNCOMMENT WHEN DISTRIBUTING', '');
+    await _regexFileAsync(filename, 'END UNCOMMENT WHEN DISTRIBUTING */', '');
+  }
+}
+
+async function _updateExpoViewAsync(packages: Package[], sdkVersion: string): Promise<void> {
+  let appBuildGradle = path.join(ANDROID_DIR, 'app', 'build.gradle');
+  let expoViewBuildGradle = path.join(ANDROID_DIR, 'expoview', 'build.gradle');
+  const settingsGradle = path.join(ANDROID_DIR, 'settings.gradle');
+  const constantsJava = path.join(
+    ANDROID_DIR,
+    'expoview/src/main/java/host/exp/exponent/Constants.java'
+  );
+  const multipleVersionReactNativeActivity = path.join(
+    ANDROID_DIR,
+    'expoview/src/main/java/host/exp/exponent/experience/MultipleVersionReactNativeActivity.java'
+  );
+
+  // Modify permanently
+  await _regexFileAsync(expoViewBuildGradle, /version = '[\d.]+'/, `version = '${sdkVersion}'`);
+  await _regexFileAsync(
+    expoViewBuildGradle,
+    /api 'com.facebook.react:react-native:[\d.]+'/,
+    `api 'com.facebook.react:react-native:${sdkVersion}'`
+  );
+  await _regexFileAsync(
+    path.join(ANDROID_DIR, 'ReactAndroid', 'release.gradle'),
+    /version = '[\d.]+'/,
+    `version = '${sdkVersion}'`
+  );
+  await _regexFileAsync(
+    path.join(ANDROID_DIR, 'app', 'build.gradle'),
+    /host.exp.exponent:expoview:[\d.]+/,
+    `host.exp.exponent:expoview:${sdkVersion}`
+  );
+
+  const filesToStash = [
+    appBuildGradle,
+    expoViewBuildGradle,
+    multipleVersionReactNativeActivity,
+    constantsJava,
+    settingsGradle,
+  ];
+  await _stashFilesAsync(filesToStash);
+
+  // Modify temporarily
+  await _regexFileAsync(
+    constantsJava,
+    /TEMPORARY_ABI_VERSION\s*=\s*null/,
+    `TEMPORARY_ABI_VERSION = "${sdkVersion}"`
+  );
+  await _regexFileAsync(
+    settingsGradle,
+    `// FLAG_BEGIN_REMOVE__UPDATE_EXPOKIT`,
+    `/*`
+  );
+  await _regexFileAsync(
+    settingsGradle,
+    `// FLAG_END_REMOVE__UPDATE_EXPOKIT`,
+    `*/ //`
+  );
+  await _uncommentWhenDistributing([appBuildGradle, expoViewBuildGradle]);
+  await _commentWhenDistributing([constantsJava, expoViewBuildGradle, multipleVersionReactNativeActivity]);
+
+  // Clear maven local so that we don't end up with multiple versions
+  console.log(' ❌  Clearing old package versions...')
+
+  await spawnAsync('rm', [
```

Comment:
`fs.remove`

Reply:
fs.remove doesn't implement `-rf`

Reply:
it does, see https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove.md